code/modules/mob/living/carbon/xenomorph/hive_status.dm

```diff
@@ -191,6 +194,26 @@
 	xeno_message(SPAN_XENOANNOUNCE("The Hive is now strong enough to support: [castes]"))
 	xeno_maptext("The Hive can now support: [castes]", "Hive Strengthening")
 
+	if(SSticker.mode.flags_round_type & MODE_INFESTATION) //checks if gamemode is a xeno gamemode before calling, will roar with no queen cause of implied queen existence
+		zlevel_evo_echo()
+
+/datum/hive_status/proc/zlevel_evo_echo()
+	for(var/mob/groundmob as anything in GLOB.player_list)
+		if(is_ground_level(groundmob.z) && !isxeno(groundmob))
+			if(initial_screech)
+				playsound_client(groundmob.client, 'sound/voice/alien_echoroar_1.ogg', groundmob.loc, 65, FALSE) //boosted a lot cause quiet audio
+				if(ishuman(groundmob))
+					to_chat(groundmob, SPAN_HIGHDANGER("You hear a distant screech and feel your insides freeze up...  something new is with you in this colony."))
+				if(issynth(groundmob))
+					to_chat(groundmob, SPAN_HIGHDANGER("You hear the distant call of an unknown bioform, it sounds like they're informing others to change form. You begin to analyze and decrypt the strange vocalization."))
+				initial_screech = FALSE
+				return
```

Comment:
Im pretty sure this will send the to chat to whatever mob is first in the list and exit the proc due to this return. You sure this is good?

Reply:
this is still a return, not a continue, by the way